pkg/apiserver/webhooks/validating-webhook/validating-webhook.go

```diff
@@ -147,6 +147,16 @@ func validateDataVolumeSpec(field *k8sfield.Path, spec *cdicorev1alpha1.DataVolu
 		return causes
 	}
 
+	if spec.Source.Registry != nil && spec.ContentType != "" && string(spec.ContentType) != string(cdicorev1alpha1.DataVolumeKubeVirt) {
+		sourceType = field.Child("contentType").String()
+		causes = append(causes, metav1.StatusCause{
+			Type:    metav1.CauseTypeFieldValueInvalid,
+			Message: fmt.Sprintf("ContentType must be KubeVirt when SourceType is Registry"),
```

Comment:
"ContentType must be KubeVirt when Source is Registry"

Reply:
I think this is what we decided. For now, I do not believe we support special 'archive' case for registry.

Reply:
:-) done